### PR TITLE
Capx 32

### PR DESCRIPTION
**Pull request by mpriest**
Feedback welcome.

### CODE REVIEW

File: stanford_capx.forms.inc [hunk 1]
```diff
@@ -709,6 +709,23 @@ function stanford_capx_admin_config_mapper_delete($form, $form_state, $machine_n
     throw new Exception("Could not load a mapper with provided machine name. Please check url.");
   }
 
+  $importers = CAPxImporter::loadImportersByMapper($mapper);
+  $items = array();
+
+  foreach ($importers as $importer) {
+    $items[] = $importer->title;
```
Comment (sherakama) [inline stanford_capx.forms.inc]: Lets link this to the edit page.
/admin/config/capx/importer/edit/[machine_name]

**Comment by sherakama** [comment]
Looks great. Just one small comment about linking the importer title to the edit page. 

Nice work.

**Comment by mpriest** [comment]
Added a link for each importer.

Ideally, I'd generate the link like this:

``` php
$edit_uri = entity_uri('capx_cfe', $importer) . '/edit';
```

But we can't do this because CFEntity->defaultUri() isn't set to work this way with the admin/config/capx/{bundle}/edit/{machine name} paths.

**Comment by sherakama** [comment]
Thanks for this Michael. I like the way it works for now. I agree with you we should change the edit/add/view paths to be consistent with best practices. I will merge this for now and create a new ticket for that work.

**Pull request merged**